src/GeneralStateTestsFiller/stCreateTest/CreateOOGafterMaxCodesizeFiller.yml

```diff
@@ -0,0 +1,347 @@
+
+          // Call for OOG contract creation
+          mstore(0, subcall_contract_count)
+          mstore(32, subcall_oog)
+          retcode := call(div(gas(), 2), 0x00000000000000000000000000000000000c0deb, 0,	0, 64, add(64, mul(delegate_contract_count, 32)), mul(subcall_contract_count, 32))
+
+          for { let i := 0 } lt(i, selfdestruct_count) { i := add(i, 1) }
+          {
+              let contract_addr := mload(add(64, mul(i, 32)))
+              pop(call(sub(gas(), 1000), contract_addr, 0, 0, 0, 0, 0))
+          }
+
+        }
+      nonce: 1
+      storage: {}
+
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '0x0ba1a9ce0ba1a9ce'
+      code: '0x'
+      nonce: '0'
+      storage: {}
+
+
+  transaction:
+    data:
+    # - Low number of contracts (10): Create 0 contracts at 0xc0dea, 10 contracts at 0xc0deb, OOG on 0xc0deb, call 0 contracts to self-destruct.
+    - :abi f(uint,uint,bool,uint) 0x00 0x0a 0x01 0x00
+    # - Low number of contracts (10): Create 10 contracts at 0xc0dea, 10 contracts at 0xc0deb, OOG on 0xc0deb, call 0 contracts to self-destruct.
+    - :abi f(uint,uint,bool,uint) 0x0a 0x0a 0x01 0x00
+    # - Low number of contracts (10): Create 10 contracts at 0xc0dea, 10 contracts at 0xc0deb, don't OOG on 0xc0deb, call 14 out of 20 contracts to self-destruct.
+    - :abi f(uint,uint,bool,uint) 0x0a 0x0a 0x00 0x0e
+    # - High number of contracts (250): Create 0 contracts at 0xc0dea, 250 contracts at 0xc0deb, OOG on 0xc0deb, call 0 contracts to self-destruct.
+    - :abi f(uint,uint,bool,uint) 0x00 0xfa 0x01 0x00
+    # - High number of contracts (250): Create 250 contracts at 0xc0dea, 250 contracts at 0xc0deb, OOG on 0xc0deb, call 0 contracts to self-destruct.
+    - :abi f(uint,uint,bool,uint) 0xfa 0xfa 0x01 0x00
+    # - High number of contracts (250): Create 250 contracts at 0xc0dea, 250 contracts at 0xc0deb, don't OOG on 0xc0deb, call 494 out of 500 contracts to self-destruct.
+    - :abi f(uint,uint,bool,uint) 0xfa 0xfa 0x00 0x01ee
+    gasLimit:
+    - 0x100000000
+    gasPrice: '10'
+    nonce: '0'
+    to: 00000000000000000000000000000000000c0dea
+    secretKey: "45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8"
+    value:
+    - 0
+
+  expect:
+
+    - indexes:
+        data: !!int 0
+        gas:  !!int -1
+        value: !!int -1
+      network:
+        - '>=London'
+      result:
+
+        00000000000000000000000000000000000c0dea:
+          nonce: 1
+          storage:
+            0x1: 0x1
+
+        00000000000000000000000000000000000c0deb:
+          nonce: 1
+          storage: {}
+
+        # 00000000000000000000000000000000000c0deb contracts:
+        # 1
+        76e2310dc9277b22ffceb40d49229131674ebe02:
+          shouldnotexist: 1
+        # 10
+        b3210b741a5dfbddc1636521965b3558defa3e60:
+          shouldnotexist: 1
+
+    - indexes:
+        data: !!int 1
```

Comment:
what I mean there is a comment in expect section to have an idea what case is happening here.

Reply:
Added comments to each expected contract.